src/storage/txn/scheduler.rs

```diff
@@ -1099,13 +1101,25 @@ impl Scheduler {
         let tag = cmd.tag();
         if readcmd {
             worker_pool.execute(move |ctx: &mut SchedContext| {
+                let process_read_timer = SCHED_PROCESSING_READ_HISTOGRAM_VEC
```

Comment:
you can use `let _timer` to avoid `drop`.  

Btw, I suggest using a local metric in the SchedContext to avoid frequent contention for the global metric. 

I also think you can remove SCHED_WORKER_COUNTER_VEC because we can use the count in SCHED_PROCESSING_READ_HISTOGRAM_VEC and SCHED_PROCESSING_WRITE_HISTOGRAM_VEC directly.

Reply:
We don't have type like `LocalHistogramVec` yet?